CONTRIBUTING.rst

```diff
@@ -0,0 +1,158 @@
+Contribution
+============
+
+This document is an extension to the generic `Anuket Project Operations and Procedures
+<https://wiki.anuket.io/display/HOME/Anuket+Project+Operations+and+Procedures>
+and describes contribution rules to the Anuket specifications, that are the set of
+documents maintained in this repository.
+
+
+Permission levels
+-----------------
+
+There are different permissions (levels) available for contributing into the Anuket specifications:
+- **Outside Collaborators**: This includes general public.
+  - Outside collaborators can create issues and add comments to issues and Pull Requests.
+- **Read, Triage, Write, and Maintain** Permissions, which are explained in
+  `here <https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/
+  repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization>,
+  will be assigned as follows:
+  - **Read** will be assigned to those who want to create issues, join the discussions, receive notifications,

```

Comment:
A blank line should be added before the second list

Reply:
@karinesevilla Thanks - Fixed